railties/test/commands/temp_test.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+require "isolation/abstract_unit"
+require "rails/command"
+
+class Rails::Command::TempTest < ActiveSupport::TestCase
+  setup :build_app
+  teardown :teardown_app
+
+  test "tmp:clear clear cache, socket, screenshot, and storage files" do
+    Dir.chdir(app_path) do
+      FileUtils.mkdir_p("tmp/cache")
+      FileUtils.touch("tmp/cache/cache_file")
+
+      FileUtils.mkdir_p("tmp/sockets")
+      FileUtils.touch("tmp/sockets/socket_file")
+
+      FileUtils.mkdir_p("tmp/screenshots")
+      FileUtils.touch("tmp/screenshots/fail.png")
+
+      FileUtils.mkdir_p("tmp/storage/6h/np")
+      FileUtils.touch("tmp/storage/6h/np/6hnp81jvgt42pcfqtlpoy8qshfb0")
```

Comment:
We can use the `app_file` helper for each of these.

Reply:
Nice. I've made the change.